src/pybind/mgr/dashboard/controllers/user.py

```diff
@@ -79,10 +113,15 @@ def set(self, username, password=None, name=None, email=None, roles=None):
             user = mgr.ACCESS_CTRL_DB.get_user(username)
         except UserDoesNotExist:
             raise cherrypy.HTTPError(404)
+        if not password and not name and not email and (user.to_dict())['roles'] == roles:
```

Comment:
Is this change related to enforcing a certain password length/complexity? I don't think that name and email have been considered mandatory so far.

Reply:
ok, I will remove it

Reply:
The problem here is that it user can be edited without changing anything, therefore I thought it would be good to add checking whether all fields are unchanged. I will remove it from this PR